tests/unit-core/test-symbol.c

```diff
@@ -0,0 +1,197 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript.h"
+#include "jerryscript-port.h"
+#include "jerryscript-port-default.h"
+#include "test-common.h"
+
+/* foo string */
+#define STRING_FOO ((const jerry_char_t *) "foo")
+
+/* bar string */
+#define STRING_BAR ((const jerry_char_t *) "bar")
+
+/* Symbol(bar) desciptive string */
+#define SYMBOL_DESCIPTIVE_STRING_BAR "Symbol(bar)"
+
+int
+main (void)
+{
+  jerry_init (JERRY_INIT_EMPTY);
+
+  if (!jerry_is_feature_enabled (JERRY_FEATURE_SYMBOL))
```

Comment:
I've just realized that there is no need to call `jerry_init` before this as the feature enable check does not depend on any runtime information. Maybe we should update the docs in a different PR to state when it is possible to call the `jerry_is_feature_enabled` function.